subsys/bluetooth/controller/hci/hci.c

```diff
@@ -3473,16 +3485,28 @@ static void le_set_per_adv_param(struct net_buf *buf, struct net_buf **evt)
 		return;
 	}
 
+	max_interval = sys_le16_to_cpu(cmd->max_interval);
+
+	if (IS_ENABLED(CONFIG_BT_CTLR_PARAM_CHECK)) {
+		uint32_t min_interval = sys_le16_to_cpu(cmd->min_interval);
+
+		if ((min_interval > max_interval) ||
+		    (min_interval < 0x0006) ||
+		    (max_interval > 0x2000)) {
```

Comment:
Let's add `#define`s in `hci.h` for these as well (or use the ones defined by `gap.h`: 
```c
/** Minimum Periodic Advertising Interval (N * 1.25 ms) */
#define BT_GAP_PER_ADV_MIN_INTERVAL             0x0006
/** Maximum Periodic Advertising Interval (N * 1.25 ms) */
#define BT_GAP_PER_ADV_MAX_INTERVAL             0xFFFF
```

Same question as to why the max is `0x2000` instead of `0xFFFF` as defined by the spec: 
![image](https://user-images.githubusercontent.com/2520413/165914130-1e18b808-ee14-4c8e-b047-7d1978055311.png)